### PR TITLE
remove nuget package installer prompts

### PR DESCRIPTION
**Pull request by jmarolf**
This fixes the following accessibility issues:

- [AzDO#1049310](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems/edit/1049310)
- [AzDo#1049313](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems/edit/1049313)
- [AzDo#1049357](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems/edit/1049357)
- [AzDo#1049358](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems/edit/1049358)

Because these analyzers are now included with the .NET SDK and we've had low usage of this feature I think the best thing to do for customers is to remove the option to install a nuget package from the code analysis property page.

## BEFORE

image

## AFTER

image

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/project-system/pull/6414)

### CODE REVIEW

**Comment by jmarolf** [comment]
CC: @vatsalyaagrawal and @jjmew this is a fix I would like to take through QB mode for 16.7

**Comment by tmeschter** [comment]
@jmarolf Can you paste in before and after screen shots so we can see what's changed?

File: src/Microsoft.VisualStudio.Editors/PropPages/CodeAnalysisPropPage.vb [hunk 1]
```diff
@@ -78,178 +47,15 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
...
-            Dim task = searchProvider.CreateSearch(dwCookie:=1, pSearchQuery:=SearchQuery.FxCopAnalyzersQueryInstance, pSearchCallback:=SearchQuery.FxCopAnalyzersQueryInstance)
-            task.Start()
-        End Sub
-
         Private Sub RoslynAnalyzersLabel_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles RoslynAnalyzersHelpLinkLabel.LinkClicked
             RoslynAnalyzersHelpLinkLabel.LinkVisited = True
             Process.Start(RoslynAnalyzersDocumentationLink)
         End Sub
 
         Protected Overrides Sub OnPageActivated(activated As Boolean)
```
Comment (ryzngard) [inline src/Microsoft.VisualStudio.Editors/PropPages/CodeAnalysisPropPage.vb:55]: nit: No need to override this anymore

**Comment by davkean** [comment]
tag @mavasani

File: src/Microsoft.VisualStudio.Editors/PropPages/CodeAnalysisPropPage.vb [hunk 2]
```diff
@@ -1,41 +1,23 @@
...
         Inherits PropPageUserControlBase
 
-        Private Const NugetFeed As String = "https://packages.nuget.org/api/v2"
-        Private Const FxCopAnalyzersPackageId As String = "Microsoft.CodeAnalysis.FxCopAnalyzers"
-        Private Const NuGetPackageManagerPackageGuid As String = "5fcc8577-4feb-4d04-ad72-d6c629b083cc"
-        Private Const NuGetPackageManagerSearchProviderGuid As String = "042C2B4B-C7F7-49DB-B7A2-402EB8DC7892"
         Private Const RoslynAnalyzersDocumentationLink As String = "https://docs.microsoft.com/visualstudio/code-quality/roslyn-analyzers-overview"
 
         Private Shared ReadOnly s_latestStableVersion As Version = New Version(2, 9, 6)
         Private Shared ReadOnly s_childPackageIds As HashSet(Of String) = New HashSet(Of String)(
```
Comment (davidwengier) [inline src/Microsoft.VisualStudio.Editors/PropPages/CodeAnalysisPropPage.vb:13]: Looks like these two can be removed too

File: src/Microsoft.VisualStudio.Editors/PropPages/CodeAnalysisPropPage.vb [hunk 3]
```diff
@@ -46,19 +28,6 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
         Protected Overrides Sub PostInitPage()
             MyBase.PostInitPage()
```
Comment (davidwengier) [inline src/Microsoft.VisualStudio.Editors/PropPages/CodeAnalysisPropPage.vb:30]: Can remove

**Comment by jjmew** [comment]
@jmarolf  and @vatsalyaagrawal  - Can you do a QB bar check on this? if the QB approves then we can merge once the insertion for Austen's change get merged (https://github.com/dotnet/project-system/pull/6412)

**Pull request merged**